external/runtime/runtime.depproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
```

Comment:
Can this be part of the test tree and restored as part of runtest (same way we restore CoreCLR/CoreFX tests there)? (I.e. make it specific to running the R2R tests, as opposed to a build dependency of the general build). The XUnit tests run as part of the build, not as part of runtests.cmd, and that's why testhost restore is there, at least for now.

I would really like to avoid downloading another copy of CoreCLR as part of the build. We already download a lot of other garbage (such as 35 MB of LibLLVM) for build configs that shouldn't care about those things. We use `build skiptests` in several configurations within the CI and those are not going to run R2R tests anyway.

Reply:
I'll fiddle with this and see if I can have it run from within runtest.cmd when at least one r2r test will get run